docs/reference/search/multi-search.asciidoc

```diff
@@ -86,6 +86,16 @@ The msearch's `max_concurrent_searches` request parameter can be used to control
 the maximum number of concurrent searches the multi search api will execute.
 This default is based on the number of data nodes and the default search thread pool size.
 
+The request parameter `max_concurrent_shard_requests` can be used to control the
+maximum number of concurrent shard requests the each sub search request will execute.
+This parameter should be used to protect a single request from overloading a cluster
+(e.g., a default request will hit all indices in a cluster which could cause shard request rejections
+if the number of shards per node is high). This default is based on the number of
+data nodes in the cluster but at most `256`. In certain scenarios parallelism isn't achieved
+through concurrent request such that this protection will result in poor performance. For
+instance in an environment where a cluster is formed by a single node holding a large amount of
+shards queried at the same time. In such a scenario it makes sense to increase the number to a high value.
```

Comment:
maybe be explicit and mention that there are never two requests running at the same time too

Reply:
Not sure I follow, you mean being explicit that this makes sense only in a single request environment? I think the kibana usecase in general makes sense where concurrently executing request are low but potentially higher than one?

Reply:
Yes this is what I meant. I'm fine with relaxing it to say low numbers of concurrent requests if that works better for you?

Reply:
++